From 1ad871c1d6dbe1b1bfa26e44562c20973888c350 Mon Sep 17 00:00:00 2001
From: Pavel Grunt <pgrunt@redhat.com>
Date: Tue, 14 Jul 2015 10:16:50 +0200
Subject: [PATCH] Remove unnecessary parameter from virt_viewer_window_resize()

Since c3cbdef888cc823e92140027d38378f40ccd4174 virt_viewer_window_resize
is always called with keep_win_size = FALSE.

(cherry picked from commit c645a99267eda3a7834b83ced428aa74c4a56e13)
---
 src/virt-viewer-window.c | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/src/virt-viewer-window.c b/src/virt-viewer-window.c
index 1d62c82..86291b8 100644
--- a/src/virt-viewer-window.c
+++ b/src/virt-viewer-window.c
@@ -67,7 +67,7 @@ void virt_viewer_window_menu_view_release_cursor(GtkWidget *menu, VirtViewerWind
 /* Internal methods */
 static void virt_viewer_window_enable_modifiers(VirtViewerWindow *self);
 static void virt_viewer_window_disable_modifiers(VirtViewerWindow *self);
-static void virt_viewer_window_resize(VirtViewerWindow *self, gboolean keep_win_size);
+static void virt_viewer_window_resize(VirtViewerWindow *self);
 static void virt_viewer_window_toolbar_setup(VirtViewerWindow *self);
 static GtkMenu* virt_viewer_window_get_keycombo_menu(VirtViewerWindow *self);
 static void virt_viewer_window_get_minimal_dimensions(VirtViewerWindow *self, guint *width, guint *height);
@@ -364,7 +364,7 @@ virt_viewer_window_desktop_resize(VirtViewerDisplay *display G_GNUC_UNUSED,
         self->priv->desktop_resize_pending = TRUE;
         return;
     }
-    virt_viewer_window_resize(self, FALSE);
+    virt_viewer_window_resize(self);
 }
 
 
@@ -412,7 +412,7 @@ virt_viewer_window_queue_resize(VirtViewerWindow *self)
  * scale down to fit, maintaining aspect ratio
  */
 static void
-virt_viewer_window_resize(VirtViewerWindow *self, gboolean keep_win_size)
+virt_viewer_window_resize(VirtViewerWindow *self)
 {
     GdkRectangle fullscreen;
     GdkScreen *screen;
@@ -475,8 +475,7 @@ virt_viewer_window_resize(VirtViewerWindow *self, gboolean keep_win_size)
     virt_viewer_display_set_desktop_size(VIRT_VIEWER_DISPLAY(priv->display),
                                          width, height);
 
-    if (!keep_win_size)
-        virt_viewer_window_queue_resize(self);
+    virt_viewer_window_queue_resize(self);
 }
 
 static void
@@ -1374,7 +1373,7 @@ virt_viewer_window_show(VirtViewerWindow *self)
     gtk_widget_show(self->priv->window);
 
     if (self->priv->desktop_resize_pending) {
-        virt_viewer_window_resize(self, FALSE);
+        virt_viewer_window_resize(self);
         self->priv->desktop_resize_pending = FALSE;
     }
 
